### PR TITLE
Handle environment errors

### PR DESCRIPTION
**Pull request by scottmmjackson**
- Handle environment errors

### CODE REVIEW

**Comment by scottmmjackson** [comment]
Guess I could have said `if 'error' in environment` but /shrug

File: rsconnect/actions.py [hunk 1]
```diff
@@ -279,6 +279,8 @@ def deploy_jupyter_notebook(connect_server, file_name, extra_files, new=False, a
     app_id, deployment_name, deployment_title, app_mode = \
         gather_basic_deployment_info(connect_server, app_store, file_name, new, app_id, title, static)
     python, environment = get_python_env_info(file_name, python, compatibility_mode, force_generate)
+    if environment.get('error', None) is not None:
```
Comment (jskress) [inline rsconnect/actions.py]:
```suggestion
    if 'error' in environment:
```

Your suggestion anyway and it reads a lot cleaner.

File: rsconnect/main.py [hunk 2]
```diff
@@ -354,6 +354,8 @@ def deploy_notebook(name, server, api_key, static, new, app_id, title, python, c
 
     with cli_feedback('Inspecting python environment'):
         python, environment = get_python_env_info(file, python, compatibility_mode, force_generate)
+        if environment.get('error', None) is not None:
```
Comment (jskress) [inline rsconnect/main.py]:
```suggestion
        if 'error' in environment:
```

File: rsconnect/main.py [hunk 3]
```diff
@@ -500,6 +502,8 @@ def manifest_notebook(force, python, compatibility_mode, force_generate, verbose
         python = which_python(python)
         environment = inspect_environment(python, dirname(file), compatibility_mode=compatibility_mode,
                                           force_generate=force_generate)
+        if environment.get('error', None) is not None:
```
Comment (jskress) [inline rsconnect/main.py]:
```suggestion
        if 'error' in environment:
```

**Pull request merged**